uk.co.compucorp.civicrm.hrleaveandabsences/views/my-leave/request-modal/details/toil.html

```diff
@@ -0,0 +1,51 @@
+<div class="col-xs-12 form-horizontal" ng-show="!$ctrl.uiOptions.multipleDays">
+  <div class="form-group">
+    <label class="col-xs-4 control-label">Date Worked</label>
+    <div class="col-xs-4">
+      <div class="input-group">
+        <input type="text" class="form-control" is-open="$ctrl.uiOptions.date.to.dateWorked"/>
```

Comment:
You can use the following code to show proper datepicker
```html
<input
          type="text"
          class="form-control"
          uib-datepicker-popup
          datepicker-options="$ctrl.uiOptions.date.dateWorked.options"
          show-button-bar="false"
          ng-model="$ctrl.uiOptions.workedDate"
          is-open="$ctrl.uiOptions.date.dateWorked.show"/>
        <span class="input-group-btn">
            <button
              type="button"
              class="btn btn-default fa fa-calendar"
              ng-click="$ctrl.uiOptions.date.dateWorked.show = true"></button>
          </span>
```

Please define `dateWorked` and `workedDate` in the controller with proper initial value. Same goes for `duration`datepickers.